osi_detectedlane.proto

```diff
@@ -24,4 +25,47 @@ message DetectedLane
     // Estimated probability that this lane really exists. Range [0,1].
     //
     optional double existence_probability = 3;
+
+    // The measurement state of the detected lane.
+    //
+    optional DetectedTrafficSign.MeasurementState measurement_state = 4;
```

Comment:
I think if the message is to be reused, it should be separated out from the DetectedTrafficSign message (it probably makes sense to have this centrally defined; I can provide a patch for this if agree upon). @CarloVanDriestenBMW ?

Reply:
See my comments on PR #117 , I'd propose (and do this during merging) to move MeasurementState into DetectedObjectHeader, and move DetectedObjectHeader out of DetectedObject, which should handle all of this cleanly, I think. @CarloVanDriestenBMW ?